### ISSUE TITLE
SUGAR, SUGAR game - sugar can't pass through the color filter

### ISSUE THREAD

**Issue by perelowski** [ruffle-rs/ruffle#14612]
### Describe the bug

In the Sugar, Sugar game - https://funkypotato.com/sugar-sugar/ - in the 3rd level the sugar has to be sent through the red filter to make it red. In the Ruffle version, it cannot pass the filter and is blocked by it.

### Expected behavior

Sugar should freely pass through the red filter just like in this gameplay video: https://youtu.be/kbEEU_t4AxM?si=XjEGMcvc9Q6qj04F&t=778

### Content Location

https://funkypotato.com/sugar-sugar/

### Affected platform

Desktop app

### Operating system

Mac OS 14.2.1

### Browser

Chrome, Firefox

**Comment by n0samu** [comment]
Duplicate of #8735
